### PR TITLE
Add createTicket endpoint and relevant services

### PR DESCRIPTION
**Pull request by ryonryon**

### CODE REVIEW

File: src/controllers/__tests__/ticket.test.ts [hunk 1]
```diff
@@ -3,49 +3,152 @@ import { build } from "../../../tests/helper";
...
-    const mockName = "this is mocked name";
-    const res = await app.inject({
-      method: "put",
-      url: "/tickets/1",
-      payload: {
-        name: mockName,
-      },
-    });
+  describe("createTicket", () => {
+    test.todo("Passed appropriate query and body - it should successfully create new ticket");
```
Comment (ryonryon) [inline src/controllers/__tests__/ticket.test.ts:7]: here I will write after test db is ready

File: src/services/__tests__/ticket.test.ts [hunk 2]
```diff
@@ -1,10 +1,48 @@
 import { build } from "../../../tests/helper";
-import { Ticket } from "../../entities";
+import { Column, Project, Ticket, TicketInput } from "../../entities";
 import ticket from "../ticket";
 
 // make a connection to real server
 build();
 
+describe("createTicket()", () => {
+  test.todo("passed proper request body - should return an created body");
```
Comment (ryonryon) [inline src/services/__tests__/ticket.test.ts:9]: here I will write after test db is ready

File: src/controllers/__tests__/ticket.test.ts [hunk 3]
```diff
@@ -3,49 +3,152 @@ import { build } from "../../../tests/helper";
 const app = build();
```
Comment (ryonryon) [inline src/controllers/__tests__/ticket.test.ts:3]: I think we don't need to connect to DB to test controllers. Let's discuss this later.

File: src/server.ts [hunk 4]
```diff
@@ -14,6 +14,7 @@ const initializeServer = () => {
   server.put("/project/:projectId", project.editProjectById);
 
   // ticket
+  server.post("/projects/:projectId/columns/:columnId/tickets", ticket.createTicket);
```
Comment (ryonryon) [inline src/server.ts:17]: We may want to give projectId and columnId in the body. Let's discuss this later.

**Review by kei95 — APPROVED** [review]

**Review by kei95 — APPROVED** [review]

**Pull request merged**